guide/creating-your-bot/command-handling.md

```diff
@@ -1,6 +1,8 @@
 # Command handling
```

Comment:
I don't agree with the changes on this page at all 😕 It ends up being much longer than it needs to be